app/views/play/level/modal/CourseVictoryComponent.vue

```diff
@@ -272,7 +272,7 @@
     
     .combo-img
       position: absolute
-      max-block-size: 70px
+      width: 70px
```

Comment:
`max-block-size` is experimental and [doesn't work on all browsers](https://developer.mozilla.org/en-US/docs/Web/CSS/max-block-size#Browser_compatibility).